NEWS

```diff
@@ -23,6 +23,8 @@ Unreleased (2.0.0)
 * `mongoc_cursor_is_alive` is removed. Use the equivalent `mongoc_cursor_more` instead.
 * `mongoc_collection_delete` is removed. Use `mongoc_collection_delete_one` or `mongoc_collection_delete_many` instead.
 * `mongoc_delete_flags_t` and `mongoc_reply_flags_t` are removed.
+* Support for `ENABLE_SASL=CYRUS` on Windows is removed. Use `ENABLE_SASL=SSPI` or `ENABLE_SASL=OFF` instead.
```

Comment:
Is there a need to support disabling SASL on Windows if SSPI ships with Windows?

Reply:
I am not sure if there is a compelling use-case, but it could be used to avoid linking to unnecessary Windows DLLs (e.g. with `ENABLE_SASL=OFF` and `ENABLE_SSL=OFF`, I see SECURE32.DLL, CRYPT32.DLL, and BCRYPT.DLL are no longer shown in Dependency Walker.